setup_atos.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+#####################################
+###### Pre-installation checks ######
+#####################################
+
+source "scripts/install_functions.sh"
+
+# Get this file location
+REPO_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+
+# Check if running on Ubuntu
+if ! grep -q "Ubuntu" /etc/os-release; then
+    echo "This script is designed for Ubuntu systems only."
+    exit 1
+fi
+
+# Set ROS_DISTRO based on Ubuntu distribution
+case "$(get_ubuntu_codename)" in
+    "focal")
+        export ROS_DISTRO=foxy
+    ;;
+    "jammy")
+        export ROS_DISTRO=humble
+    ;;
+    *)
```

Comment:
Is this really correct? I think that you can have ubuntu 20 and humble for example.. Would this case break things further downstream?

Reply:
The ROS_DISTRO env variable is set/will be overwritten when sourcing /opt/ros/distro/setup.zsh
If a ros distro is already in use then running this script might overwrite this variable in the active environment of the user. Is there a reason for overwriting actual environment variables such as this one and NVM_HOME, or could temporary "script-local" variables be used (and cleaned up afterwards)?

Reply:
I took (wrongfully) for granted that that ROS distros where basically tied to Ubuntu distros. Since they are not, I removed the Foxy support since we only have one user running foxy still.